src/main/java/com/bankbazaar/test/entity/Department.java

```diff
@@ -0,0 +1,24 @@
+package com.bankbazaar.test.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Table(name = "Department")
```

Comment:
Table names should begin with a lower case letter

Reply:
THis issue has been resolved.